src/Logging/ServiceCollectionExtensions.cs

```diff
@@ -29,23 +31,32 @@ public static IServiceCollection AddOmexServiceContext<TServiceContext>(this ISe
 		/// Adds Omex event logger to the factory
 		/// </summary>
 		/// <param name="builder">The extension method argument</param>
-		public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder)
+		/// <param name="context">HostBuilderContext to access Configuration</param>
+		public static ILoggingBuilder AddOmexLogging(this ILoggingBuilder builder, HostBuilderContext? context)
 		{
-			builder.Services.AddOmexLogging();
+			builder.Services.AddOmexLogging(context);
 			return builder;
 		}
 
 		/// <summary>
 		/// Adds Omex event logger to the factory
 		/// </summary>
 		/// <param name="serviceCollection">The extension method argument</param>
+		/// <param name="context">HostBuilderContext to access Configuration</param>
 		/// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained</returns>
-		public static IServiceCollection AddOmexLogging(this IServiceCollection serviceCollection)
+		public static IServiceCollection AddOmexLogging(this IServiceCollection serviceCollection, HostBuilderContext? context)
 		{
-			serviceCollection.AddLogging();
-
 			serviceCollection.TryAddTransient<IServiceContext, EmptyServiceContext>();
 			serviceCollection.TryAddTransient<IExecutionContext, BaseExecutionContext>();
+			serviceCollection.AddLogging();
+
+			const string settingName = "Monitoring:OmexLoggingEnabled";
+			bool isEventSourceLoggingEnabled = bool.Parse(context?.Configuration.GetSection(settingName).Get<string>() ?? "true");
```

Comment:
Not sure if this works exactly like this, but I would expect something simular to this code should be possible, could you please check?
```suggestion
			bool isEventSourceLoggingEnabled = context?.Configuration.GetSection(settingName).Get<bool>(true);
```